src/Microsoft.IdentityModel.Abstractions/DataClassification.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.IdentityModel.Abstractions
+{
+    /// <summary>
+    /// Data classification categories defined by the asset classification standard.
+    /// </summary>
+    /// <remarks>
+    /// See http://aka.ms/AssetClassificationStandard for more details about the asset classification standard.
+    /// </remarks>
+    public enum DataClassification
+    {
+        /// <summary>
+        /// Data used to manage access to administrative roles or sensitive functions.
+        /// </summary>
+        AccessControlData = 0,
+
+        /// <summary>
+        ///This is the data, information and code that C1 admins, C1 users, and C2 users (if present) provide to, transfers in, stores in or
+        /// process in the Microsoft online service or product.
+        /// </summary>
+        /// <remarks>The term C1 refers to Microsoft’s customer.The term C2 refers to the customer’s customer.</remarks>
+        CustomerContent = 10,
```

Comment:
I assume these were copied from MISE / Office ? Is there a way to combine 2 of these levels (like a flags enum) ?

Reply:
Also, from the design guidelines for Flags enum: 

```
Designing Flag Enums

✔️ CONSIDER providing special enum values for commonly used combinations of flags.

```

So maybe we could have an `AnyPii` value for this enum, as most of our customers are not used to this kind of classification.

Reply:
I envisioned it as listing the highest exposed classification. e.g. if both PII and OII then the item is PII

Reply:
Ack, so this is a not a flags enum, i.e. app developers can't pick and choose several options, just one. Kind of like logging verbosities.

Reply:
I don't believe it's a design goal to allow for extensibility here. It doesn't seem very limiting to say a dimension will only be classified by one of these. Can you give an example where one should have 2 classifications? Seems like that's generally an argument for 2 dimensions...